src/interfaces/SpokePool.ts

```diff
@@ -57,6 +57,7 @@ export interface RelayExecutionEventInfo {
 }
 
 export interface Fill extends RelayData {
```

Comment:
todo: Should also drop `message` from this type, and from `SlowFillRequest`. That will flush out a few extra incompatibilities.